### ISSUE TITLE
Add Installation Instructions

### ISSUE THREAD

**Issue by FrederickGeek8** [Introtocs/GitHub-Training-Matplotlib#1]
I'll probably submit a PR later, but at least for me (macOS) I needed to run the following command to get the prerequisites installed.
```bash
sudo pip3 install matplotlib
```

**Comment by shikhar394** [comment]
Yeah, this is great. Send in a pull request if you want but it is quite likely that this won't be the official group!
Thanks!
